multicluster/build/yamls/antrea-multicluster-leader-namespaced.yml

```diff
@@ -456,6 +457,7 @@ kind: ValidatingWebhookConfiguration
 metadata:
   labels:
     app: antrea
+    served-by: antrea-mc
```

Comment:
This is generated automatically, you should add a common label in the file `antrea/multicluster/config/webhook/kustomization.yaml` like following:
```
commonLabels:
  served-by: antrea-mc-controller
```
Then run `make manifests` in the `multicluster` sub-folder to update these manifests.

Reply:
updated.